### ISSUE TITLE
Copy CID/URL issues

### ISSUE THREAD

**Issue by alanshaw** [nftstorage/nft.storage#1145]
<img width="663" alt="Screenshot 2022-01-24 at 10 18 29" src="https://user-images.githubusercontent.com/152863/150765207-627c79a6-e757-4c66-a483-830ee7890e57.png">

- "cid" should be all caps ("CID") in tooltip

<img width="751" alt="Screenshot 2022-01-24 at 10 18 36" src="https://user-images.githubusercontent.com/152863/150765307-e5279c45-e7b6-4a87-ad43-fd5d5faccd8c.png">

-  "Url" should be all caps ("URL") in tooltip
- URL should not be in bold

<img width="814" alt="Screenshot 2022-01-24 at 10 23 17" src="https://user-images.githubusercontent.com/152863/150765474-c4c50d56-1ca2-42e8-9c1f-2f57186bd398.png">

* Some URLs link to the gateway, some link to ipfs://...
* For "NFT" type uploads, please can you check the URL _still_ links to `ipfs://<CID>/metadata.json`?
* I'm not convinced by the UX of`URL: <copy icon>`. The copy icon means copy the text you can see - the text "URL:" is not what we want to copy. I'm not a fan of the repetition "URL:" either. Another proposal - toggle icon in column header which toggles between CID/URL and then just one copy icon next to each CID/URL. I [suggested something slightly different here also](https://github.com/nftstorage/nft.storage/pull/1085#issuecomment-1016340526). Please can we implement one of these or get a designer involved to come up with another alternative we're happy with?

---

Separately:

@dchoi27 I'm not convinced "copy" for the sake of adding 7 charatcers ("ipfs://") to the front the the CID is worth it...is this the right place (and time(?)) to be making it hard for people to view NFTs? I agree we shouldn't do this in the client library and in our HTTP API - where people are actually using the output to mint NFTs, but when users are viewing this information on a website, in a browser, not linking to a gateway URL where they can actually view the content seems unnecessarily onerous.

**Comment by dchoi27** [comment]
I do think it's worth the investment to make it frictionless to copy an IPFS URL. It's just something we want users to consider and learn more about to help climb the big standards mountain.

But I thought this would be a super minor change (just adding copy button(s) for the CID + `ipfs://` URL). It also looks like we've removed the ability to click on the CID to open a gateway URL with the CID, which I don't think should be the case - the intent here wasn't to take out gateway links completely, it was just to add the copy options. Totally agreed that it's not the place to make it hard for folks to view their uploaded data - we need to add this back.

Fair point on the UX being confusing. I think what you suggest in the comment [here](https://github.com/nftstorage/nft.storage/pull/1085#issuecomment-1016340526) for a pop-up box that lays out the different copy options makes thinks clear. A user coming in with no background on the difference between URL types, etc. could get confused by the way it's currently laid out. My bad on that.

For now - I think rolling back #1085 could make the most sense? @JeffLowe had suggested there doing this first round of changes for an MVP implementation, then bringing in whatever UI we ended up with in the large uploader app, but the copy IPFS URL option isn't worth it given all the confusion it's caused. I think @alanshaw's suggestion for the copy interface [here](https://github.com/nftstorage/nft.storage/pull/1085) should be considered for the large uploader interface, cc @jchris.

Should definitely keep note of the typos that Alan flags when/wherever this does get implemented.
